catalystwan/models/configuration/feature_profile/sdwan/embedded_security/ngfirewall.py

```diff
@@ -716,6 +842,37 @@ class Sequence(BaseModel):
         serialization_alias="disableSequence",
     )
 
+    def add_log_action(self) -> None:
+        if LogAction in map(type, self.actions):
+            raise ValueError("LogAction is already added to NGfirewall sequence")
```

Comment:
can we move this check to `actions` validator as an assertion so `ValidationError` will be used which i believe is correct type for such exception (Also convert results are validated again after conversion and there is already a mechanism to report such errors)